README.md

```diff
@@ -1,6 +1,28 @@
-# Hyperchains Contract
+Hyperchains Contract
+====================
+
+Proxy lib for the AEternity project. When used as a dependency, provides compiled and ready to use StakingContract.json
+
+The source code (in Sophia programming language) for the contract is in `/priv/SimpleElection.aes`
```

Comment:
Make this a clickable link like this: `[/contarcts/Election.aes](contracts/Election.aes)`